GraphicEditor/View/UserControls/CardiologyFirstFloorMapUserControl.xaml

```diff
@@ -0,0 +1,450 @@
+﻿<UserControl x:Class="GraphicEditor.View.UserControls.CardiologyFirstFloorMapUserControl"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             mc:Ignorable="d" 
+             xmlns:vml="clr-namespace:GraphicEditor.VML"
+             vml:ViewModelLocator.AutoHookedUpViewModel="True"
+             d:DesignHeight="650" d:DesignWidth="1200">
+
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <!-- #Styles -->
+                <ResourceDictionary Source="pack://application:,,,/Resources/Styles/ButtonStyles.xaml" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <Border BorderBrush="Black" 
+            BorderThickness="1">
+        <ScrollViewer>
+            <Grid>
+                <!--FloorPattern-->
+                <Image Height="600" 
+                       Width="672"
+                       HorizontalAlignment="Center"
+                       Source="/GraphicEditor;component/Resources/Images/floor.jpg" 
+                       VerticalAlignment="Center"/>
+
+                <!--Room1 button-->
+                <Button x:Name="Room1Button"
+                        Background="#FF237793" 
+                        BorderBrush="Black"
+                        CommandParameter="room1"
+                        Height="Auto"
+                        Margin="254,122,763,453"
+                        Style="{StaticResource ResourceKey=DefaultBuildingButtonStyle}">
```

Comment:
Background, Height, and BorderBrush is the same in every button. I suggest you to make a new style in Styles.xaml with BasedOn property with value DefaultBuildingButtonStyle, and move these repeating properties there. By doing this you will significantly shorten the code.